packages/build/package.json

```diff
@@ -29,18 +29,18 @@
 	},
 	"homepage": "https://github.com/cydran/cydran#readme",
 	"dependencies": {
-		"@babel/preset-env": "^7.18.2",
-		"@babel/preset-typescript": "^7.17.12",
-		"@jest/globals": "^28.1.1",
-		"@types/jest": "28.1.3",
-		"express": "4.17.2",
-		"fs-extra": "10.0.0",
-		"jest-cli": "^28.1.1",
-		"jest-environment-jsdom": "^28.1.1",
-		"ts-jest": "^28.0.5",
-		"typescript": "4.7.4"
+		"@babel/preset-env": "^7.25.4",
+		"@babel/preset-typescript": "^7.24.7",
```

Comment:
These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

Reply:
fixing